### ISSUE TITLE
Option → Templates needs scrollbar.

### ISSUE THREAD

**Issue by ParkJaeSeong** [coddingtonbear/obsidian-web#22]
Thanks for the amazing expansion. Finally, it is very easy to get information on the web in obsidian.
When I modify template, I can't save changes because the "save change" button moves to the bottom of browser.

![image](https://user-images.githubusercontent.com/9565700/176170272-190d1362-f3f8-4efc-a0ff-8a0ee2e89ffd.png)

**Comment by skouzini** [comment]
This is by no means a permanent fix, but here is a workaround that I made:

1. Open 'Obsidian Web Options'
2. Select the pencil icon for the template you want to edit
3. After modal renders, open Chrome DevTools (`F12` for me is the fastest way)
4. Go to the Console tab and enter this code:
```js
let canvasNode = document.querySelector(".MuiModal-root")
canvasNode.setAttribute("style", canvasNode.getAttribute("style")+"margin: 50px 100px; padding: 25px; overflow-y: scroll;")
```
5. You should be able to close DevTools and do what you needed to do. If you close that page, you will have to redo this process, as it is not a persistent workaround. 

Hopefully this will help!

**Comment by coddingtonbear** [comment]
Incidentally, this is already sorted out in the main branch (see https://github.com/coddingtonbear/obsidian-web/commit/36e5b6e3de94f087ffc030c5aab5af274019d5b8) -- I just haven't had time to package up a new release.

**Comment by wrgrant** [comment]
It is so fun to discover a bug, then find the issue has already been reported, and even fixed in this case!!

**Comment by coddingtonbear** [comment]
I've just submitted a build to the chrome extension store including the fix mentioned earlier (36e5b6e); it might be a few days before they get around to reviewing/approving/releasing the new package, but this problem should go away soon.  I'm going to close this issue now that the fix is in the pipeline, though.  Good luck!
